### PR TITLE
correct table quoting and prefixing in migration

### PR DESCRIPTION
**Pull request by pistej**

### CODE REVIEW

**Comment by machour** [comment]
Thank you !

File: migrations/m160509_211405_add_flashed_to_notification.php [hunk 1]
```diff
@@ -6,11 +6,11 @@ class m160509_211405_add_flashed_to_notification extends Migration
 {
     public function up()
     {
-        $this->addColumn('notification', 'flashed', $this->boolean()->notNull());
+        $this->addColumn('{{%notification}}', 'flashed', $this->boolean()->notNull());
```
Comment (michaelst) [inline migrations/m160509_211405_add_flashed_to_notification.php]: What is the purpose in doing the table name this way?

Comment (pistej) [inline migrations/m160509_211405_add_flashed_to_notification.php]: this table name is more general, independent on database type (quoting) and table prefix
[http://www.yiiframework.com/doc-2.0/guide-db-dao.html#quoting-table-and-column-names](http://www.yiiframework.com/doc-2.0/guide-db-dao.html#quoting-table-and-column-names)

**Comment by machour** [comment]
@michaelst it enables you to prefix your tables by configuring \yii\db\Connection
See this attribute doc: http://www.yiiframework.com/doc-2.0/yii-db-connection.html#$tablePrefix-detail

Use case : I'm on a hosted server with only one database allowed. I want to host two distinct yii apps and they both contain a 'user' table. Without the prefix I'm screwed. With the prefix, I can configure my first app to use "app1_" as prefix, and the second one "app2_". Resulting tables would be app1_user & app2_user.

**Pull request merged**